config/module.config.php

```diff
@@ -1,33 +1,33 @@
-<?php
-
-use Zend\Db\Adapter\Adapter;
-
-return array(
-
-	'controllers' => array(
-		'invokables' => array(
-			'IvantageJobQueue\Controller\JobQueue' => 'IvantageJobQueue\Controller\JobQueueController'
-		)
-	),
-	
-	'router' => array(
-		'routes' => array(
-			'job-queue' => array(
-				'type' => 'literal',
-				'options' => array(
-					'route' => '/jobqueue',
-					'defaults' => array(
-                        'controller' => 'IvantageJobQueue\Controller\JobQueue',
-                        'action'     => 'jobQueue',
-                    )
-				)
-			)
-		)
-	),
-
-	'view_manager' => array(
-		'strategies' => array(
-			'ViewJsonStrategy'
-		)
-	)
+<?php
+
+use Zend\Db\Adapter\Adapter;
+

```

Comment:
This is where zend/db is used but on a closer look it doesn't seem to be used

Reply:
Thanks! Yeah, looks like that isn't used. We can go ahead and remove the `use` and the `zend-db` package requirement.